plugins/asyncio/uwsgiplugin.py

```diff
@@ -1,10 +1,10 @@
-from distutils import sysconfig
+import sysconfig
```

Comment:
`sysconfig.get_path` also allows us to specify which scheme we want with the second argument, rather than what you've done here. However, I'm loathe to check if 'deb_system' is a suitable scheme, but this only appears to rear it's head on Python 2, rather than every version of Python 3 that also ships distutils.

```
barriere% python3
Python 3.9.2 (default, Feb 28 2021, 17:03:44) 
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sysconfig
>>> sysconfig.get_path('include')
'/usr/include/python3.9'
>>> 
barriere% python2 
Python 2.7.18 (default, Jul 14 2021, 08:11:37) 
[GCC 10.2.1 20210110] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import sysconfig
>>> sysconfig.get_path('include')
'/usr/local/include/python2.7'
>>> sysconfig.get_path('include', 'deb_system')
'/usr/include/python2.7'
```